src/js/components/ServicesTable.js

```diff
@@ -109,15 +111,27 @@ var ServicesTable = React.createClass({
     );
   },
 
-  renderHealth: function (prop, service) {
-    let appHealth = service.getHealth();
+  renderStatus: function (prop, service) {
+    let instances = service.getInstancesCount();
+    let serviceStatus = service.getStatus();
+    let serviceStatusClassSet = StatusMapping[serviceStatus] || '';
+    let tasks = service.getTasksSummary();
```

Comment:
Let's stick to the scope of this PR! The [`getTasksSummary`}(https://github.com/dcos/dcos-ui/blob/feature/unified-ui/src/js/structs/Service.js#L107) method is part of the service struct interface and is not part of this feature. Let's do this refactoring in a different branch. PRs welcome!